testing/testpb/v1/test.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package testing.testpb.v1;
 
-option go_package = ".;testpb";
+option go_package = "./;testpb";
```

Comment:
Why was this change made?

Reply:
I couldn't run `make all` without this change, see below:
<img width="1036" alt="Screenshot 2021-12-05 at 11 57 35 PM" src="https://user-images.githubusercontent.com/6687820/144753828-170ecbf6-2d44-4c88-a0fd-dcd37531fa32.png">

I guess different versions of `protoc-gen`.
Let me revert this change and push again.